pom.xml

```diff
@@ -231,7 +231,9 @@
     <!-- org.fusesource.leveldbjni will be used except on arm64 platform. -->
     <leveldbjni.group>org.fusesource.leveldbjni</leveldbjni.group>
     <kubernetes-client.version>6.13.3</kubernetes-client.version>
-    <okio.version>1.17.6</okio.version>
+    <okio.version>3.6.0</okio.version>
+    <okhttp.version>4.12.0</okhttp.version>
+    <kotlin-stdlib.version>1.9.10</kotlin-stdlib.version>
```

Comment:
This looks like outdated because this is not the latest (2.0.10) nor the latest of 1.9.x line. Is there a reason we need to be stuck with this specific version?
- https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.9.25
- https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib-jdk8/2.0.10

Reply:
Hi @dongjoon-hyun,

Thanks for the review! 

> This looks like outdated because this is not the latest (2.0.10) nor the latest of 1.9.x line. Is there a reason we need to be stuck with this specific version?

No special reason, I just used the higher version number what I have seen in the build/mvn dependency:tree command's output. 

Updated the pull request and increased the okio version to 3.9.0 and the kotlin-stdlib version to 2.0.10. I found these latest versions in the Updates column of the following page:

https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/4.12.0